config/config.js

```diff
@@ -41,8 +41,9 @@ module.exports = {
   contentStore: {
     type: process.env.CONTENTSTORE_TYPE || 'file',
     timeout: process.env.CONTENTSTORE_TIMEOUT || 5000,
-    baseUrl: process.env.CONTENTSTORE_BASEURL,
+    apiBaseUrl: process.env.CONTENTSTORE_BASEURL,
```

Comment:
Yeah, maybe we should rename the env var to make it clear what it's being used for.